components/LandingHeader/style.scss

```
@@ -0,0 +1,38 @@
+@import '../../assets/style/mixins';
```

You can use the same absolute imports in Sass, but have to prepend a `~` before.
```suggestion
@import '~@/assets/style/mixins';
```